src/SFML/Graphics/Shape.cpp

```diff
@@ -25,24 +25,25 @@
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
-#include <SFML/Graphics/Shape.hpp>
 #include <SFML/Graphics/RenderTarget.hpp>
+#include <SFML/Graphics/Shape.hpp>
```

Comment:
clang-format by default will try to include the main header file first but that seemingly only works for headers within the same directory that are included via quotes without any directory prefixes. I've tried messing around with the header sorting options to fix this but cannot figure it out.

Reply:
We can fix this by used quoted include paths but at the moment we don't want to do that for these reasons:

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#sf12-prefer-the-quoted-form-of-include-for-files-relative-to-the-including-file-and-the-angle-bracket-form-everywhere-else

Maybe we reevaluate this later.